SignalMessaging/categories/UIView+OWS.h

```diff
@@ -173,6 +173,26 @@ CG_INLINE CGSize CGSizeMax(CGSize size1, CGSize size2)
     return CGSizeMake(MAX(size1.width, size2.width), MAX(size1.height, size2.height));
 }
 
+CG_INLINE CGPoint CGPointAdd(CGPoint left, CGPoint right)
+{
+    return CGPointMake(left.x + right.x, left.y + right.y);
+}
+
+CG_INLINE CGPoint CGPointSubtract(CGPoint left, CGPoint right)
+{
+    return CGPointMake(left.x - right.x, left.y - right.y);
+}
+
+CG_INLINE CGPoint CGPointScale(CGPoint point, CGFloat factor)
+{
+    return CGPointMake(point.x * factor, point.y * factor);
+}
+
+CG_INLINE CGSize CGSizeScale(CGSize size, CGFloat factor)
+{
+    return CGSizeMake(size.width * factor, size.height * factor);
+}
```

Comment:
I've been meaning to elaborate these inline functions for a while.  They're useful for writing more readable code.